app/views/exportImport.html

```diff
@@ -0,0 +1,128 @@
+<div class="page-content">
+  <div class="row">
+    <!-- include the sidebar -->
+    <div ng-include="'views/sidebar.html'"></div>
+    <div class="col-md-10">
+      <div class="content-box-large">
+        <div class="panel-heading">
+          <div class="panel-title">
+            <h2><i class="glyphicon glyphicon-transfer"></i>&nbsp;&nbsp;Export/Import</h2>
+          </div>
+        </div>
+
+        <!-- Top Alerts -->
+        <alert ng-repeat="alert in alerts.top" type="alert.type">{{alert.msg}}</alert>
+        <!-- Top Alerts -->
+        
+        <div class="exportSectionDiv">
+
+          <h3><strong>Export Data</strong></h3>
+          <div style=" float: left; clear: both; width: 100%; margin-bottom: 10px;">
+            <label tooltip="For a 'Insert Script' all '_id' properties will be removed and re-created on import">Will this export file be used as an Insert or Update?</label><br />
```

Comment:
Maybe for a tooltip we should speak to something the user will understand. Something like: "For an 'Insert Script' the exported data will always add new elements, however, an 'Update Script' will attempt to update existing elements if possible"